src/frontend/src/lib/components/carousel/Controls.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+	import { IconWest, IconEast } from '@dfinity/gix-components';
+	import { i18n } from '$lib/stores/i18n.store';
+
+	export let onNextClick: () => void;
+	export let onPreviousClick: () => void;
+
+	const classes =
```

Comment:
What about creating a component `ButtonControl`? We generally do not have a pattern in which we assign classes as const.

Reply:
Makes sense, create a new button component as suggested.